packages/website/pages/docs/guides/build-on-taiko/setup-your-wallet.mdx

```diff
@@ -11,11 +19,45 @@ This guide helps you connect your wallet to Taiko's networks.
 ## Steps
 
 <Steps>
-    ### Visit RPC configuration
-    Visit the [RPC configuration page](/docs/reference/rpc-configuration) and click the buttons to add the networks. Or, add them manually in your wallet with the provided configuration.
+    ### Add chains to your wallet
+    <ConnectToMetamaskButton network={SEPOLIA_CONFIG.names.shortName} />
+    <br />
+    <ConnectToMetamaskButton network={GRIMSVOTN_CONFIG.names.shortName} />
+    <br />
+    <ConnectToMetamaskButton network={ELDFELL_CONFIG.names.shortName} />
 
     ### Add tokens to your wallet
-    Use your wallet (e.g., Metamask) to [import the tokens](https://support.ledger.com/hc/en-us/articles/6375103346077-Add-custom-tokens-to-MetaMask?docs=true). You can find all deployed tokens contract addresses [here](/docs/reference/contract-addresses).
+    <AddTokenButton address={SEPOLIA_CONFIG.basedContracts.erc20Contracts.taikoToken.address.proxy} 
```

Comment:
we should either add a vertical margin to the bottom or <br/> elements as it's rendering squished ![image](https://github.com/taikoxyz/taiko-mono/assets/13951458/c6a5873a-52ed-4e95-8587-2eb1d1e2c100)